package.json

```diff
@@ -13,7 +13,20 @@
     ":copy-schematic-files": "cpx \"src/**/_files/**/**\" lib",
     "parcel": "ngc -p tsconfig.parcel.json",
     "spa": "tsc -p tsconfig.spa.json",
-    "node": "tsc -p tsconfig.node.json"
+    "node": "tsc -p tsconfig.node.json",
+    "// - Integration": "Building apps that are required for E2E testing",
+    "build:integration:shop": "yarn --cwd integration/shop build:single-spa",
```

Comment:
Why the switch from npm to yarn? I don't really care whether this project uses npm or yarn, but feel pretty strongly that it should only use one or the other (not both). I personally prefer yarn, but the previous person helping out with single-spa-angular didn't like it so I let him change it over to npm.

Reply:
See comment above.